docs/README.md

```diff
@@ -5,8 +5,8 @@ The website content it is deployed automatically in the `gh-pages` branch by Git
 After cloning the repo (with submodules), just run `make serve` to test the website locally.
```

Comment:
front README have still some references to cOS-toolkit!

Reply:
wow, I been searching for those and could not find. You meant the main root README.md right? :rofl: